ua-parser.js

```diff
@@ -68,6 +68,33 @@ const parseUA = (userAgent, browsers) => {
   data.browser.name = browsers[data.browser.id].name;
   data.inBcd = false;
 
+  if (data.browser.id === 'safari') {
+    // Handle Safari backport versions
+    if (
+      compareVersions.compare(data.version, '4.1', '>=') &&
+      compareVersions.compare(data.version, '5', '<')
+    ) {
+      // Safari 4.1 is a backported version of Safari 5
```

Comment:
Can you also link to https://en.wikipedia.org/wiki/Safari_version_history#Safari_4 + https://en.wikipedia.org/wiki/Safari_version_history#Safari_5 as evidence? The release date and WebKit version is the same for 4.1 and 5.0.